### PR TITLE
[Db-771][Risk=no] Refactor modules 

### PR DESCRIPTION
**Pull request by mhigbyflowers**

### CODE REVIEW

File: public-ui/src/app/research-directory/research-directory-routing-module.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    path: 'login',
+    component: LoginComponent,
+    data: { title: 'Sign In' }
+  },
+  {
+    path: '',
+    canActivate: [SignInGuard],
+    canActivateChild: [SignInGuard],
+    runGuardsAndResolvers: 'always',
+    children: [
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/research-directory/research-directory-routing-module.ts:21]: This path is probably not needed for now.

Comment (SrushtiGangireddy) [reply public-ui/src/app/research-directory/research-directory-routing-module.ts:21]: and this routing module

File: public-ui/src/app/shared/components/breadcrumb/breadcrumb.component.html [hunk 2]
```diff
@@ -2,11 +2,19 @@
   <div class="breadcrumb">
     <a href="{{allOfUs}}/">Home</a>
   </div>
-  <div class="breadcrumb">
-    <a href="{{allOfUs}}/data/">Data</a>
-  </div>
+  <ng-container *ngIf="isResearchDirectory;else isDatabrowser">
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/shared/components/breadcrumb/breadcrumb.component.html]: Could be make databrowser default here and add research directory when we have code ?

File: public-ui/src/app/shared/components/breadcrumb/breadcrumb.component.ts [hunk 3]
```diff
@@ -12,20 +12,22 @@ export interface Breadcrumb {
...
+    '../../../styles/page.css',
+    '../../../styles/headers.css',
+    '../../../styles/inputs.css',
     './breadcrumb.component.css']
 })
 export class BreadcrumbComponent implements OnInit, OnDestroy {
   subscription: Subscription;
   breadcrumbs: Breadcrumb[];
   allOfUs = environment.researchAllOfUsUrl;
+  isResearchDirectory = environment.isResearchDirectory;
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/shared/components/breadcrumb/breadcrumb.component.ts]: Do not need this for now may be.

File: public-ui/src/environments/environment.test.ts [hunk 4]
```diff
@@ -6,5 +6,6 @@ export const environment = {
   debug: false,
   gtmId: 'GTM-NQ9XDTW',
   gtmAuth: 'kMy-ZUX8XsBfuanMZzP_5A',
-  gtmPreview: 'env-25'
+  gtmPreview: 'env-25',
+  isResearchDirectory: true
```
Comment (SrushtiGangireddy) [inline public-ui/src/environments/environment.test.ts]: We could just use isDataBrowser from here.

**Review by SrushtiGangireddy — APPROVED** [review]
Looks good. Thanks for your effort on this.
Other than some research directory code in there, rest looks great.

File: public-ui/src/environments/environment.test.ts (hunk 4 above)
Comment (mhigbyflowers) [reply public-ui/src/environments/environment.test.ts]: thanks for catching these

**Pull request merged**